Index: src/js/script.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>'use strict' ;\r\nlet result = 0;\r\nconst number = +prompt('Enter your number');\r\nif (number < 1) {\r\n    console.log(NaN)\r\n}else {\r\n     for (let i = 2; i < number - 1; i++) {\r\n         if (number % i === 0) {\r\n            result = i; break\r\n            }\r\n        }\r\n    }\r\n    if (result === 0) {\r\n        console.log('Просте число')\r\n    }else {\r\n        console.log(result)\r\n    }\r\n\r\n\r\n\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/js/script.js b/src/js/script.js
--- a/src/js/script.js	
+++ b/src/js/script.js	
@@ -1,20 +1,5 @@
 'use strict' ;
-let result = 0;
-const number = +prompt('Enter your number');
-if (number < 1) {
-    console.log(NaN)
-}else {
-     for (let i = 2; i < number - 1; i++) {
-         if (number % i === 0) {
-            result = i; break
-            }
-        }
-    }
-    if (result === 0) {
-        console.log('Просте число')
-    }else {
-        console.log(result)
-    }
+
 
 
 
